src/apps/stitcher/main.cpp

```diff
@@ -65,7 +65,7 @@ namespace {
         float s, t;
     };
 
-    constexpr std::string_view VertexShader = R"(
+    constexpr std::string_view vertexShader = R"(
```

Comment:
These should be capitalized as they were before since they are `constexpr`

Reply:
ok, needs to change the usage below then,  since that is lowercase...

Reply:
modified the usage instead